lib/rubocop/cop/style/inverse_methods.rb

```diff
@@ -175,6 +178,15 @@ def camel_case_constant?(node)
         def dot_range(loc)
           range_between(loc.dot.begin_pos, loc.expression.end_pos)
         end
+
+        def contains_jump?(node)
+          return false unless node.is_a?(RuboCop::AST::Node)
```

Comment:
one-liner:
`node.each_node(:return, :break, :next).any?`
each_node starts from the receiver (`node`) and recursively visits child nodes, with optional filtering by types (which is what you are checking), and yields only nodes, so no need to check for the class.

Reply:
ty, done!